cmd/lnmuxd/run.go

```diff
@@ -106,9 +109,10 @@ func runAction(c *cli.Context) error {
 		return err
 	}
 
-	// Instantiate grpc server.
+	// Instantiate grpc server and enable reflection and Prometheus metrics.
```

Comment:
Besides the grpc metrics, what other lnmux-specific metrics do you think are useful to export?